ephios/api/templates/api/access_token_list.html

```diff
@@ -25,6 +25,9 @@ <h5 class="card-title">
                                 {{ token.application.name }}
                             </h5>
                             <p class="text-muted">
+                                {% if token.application.federatedhost_set.exists %}
+                                    {% translate "ephios instance" %}
+                                {% endif %}
```

Comment:
I don't really like how the **plugin** federation spills over these specifics into the other app. Can we find another way to adapt the look of the application list and token list?

Reply:
(Token list probably not even needs this hint)

Reply:
> (Token list probably not even needs this hint)

I think it helps the user to understand where this is coming from

Reply:
we agreed that we will keep it but write a comment